hack/install_devconsole/README.md

```diff
@@ -1,25 +1,39 @@
-##Non-admin user viewing the console with developer console perspective
+## Developer console for non-admin users
 
-Refers [https://jira.coreos.com/browse/ODC-347](https://jira.coreos.com/browse/ODC-347)
+You can use `consoledeveloper.sh` script available in this directory
+to enable developer console for a non-admin user.
 
-This PR provides a script to install
+Before running the script, ensure you have set `KUBECONFIG`
+environment variable pointing to the Kubernetes configuration file.
 
-the latest console with the developer perspective, and
-the devconsole operator needed to enable the perspective.
-The prerequisites for testing this are
-export KUBECONFIG=kubeconfig file
+    export KUBECONFIG=/path/to/kubeconfig
 
-Run the script consoledeveloper.sh
-It does the following:
-1. Replaces the existing openshift console with the talamer console
-2. Installs the operator. (Prompts if it already exists)
-3. Creates a non-admin user consoledeveloper with the password as developer with the suitable rolebinding(rolebinding being used here is self-provisioner and view)
+Also, ensure you have the latest `oc` command available in the `PATH`
```

Comment:
Can we specify the minimum required version of _oc_ utility to be used?

Reply:
Done. 34ec7dca8cb84c02acd0133139198be6d82cfe3c